cypress/support/generate.js

```diff
@@ -0,0 +1,24 @@
+const faker = require('faker');
+
+function generateUser() {
+  const genders = ['Male', 'Female', 'Other'];
+  const subjects = ['Chemistry', 'English', 'Computer Science'];
+  const hobbies = ['Sports', 'Reading', 'Music'];
```

Comment:
Declare the arrays 'genders', 'subjects', and 'hobbies' as constants outside of the 'generateUser' function to avoid redeclaring them each time the function runs.